src/Entity/Geo/Canton.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Entity\Geo;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="geo_canton")
+ * @Algolia\Index(autoIndex=false)
+ */
+class Canton
+{
+    /**
+     * @var string
+     *
+     * @ORM\Id
+     * @ORM\Column
+     */
+    public $id;
```

Comment:
I think in the rest of the project, we are not using public properties for entities, maybe we should set them private and use getter/setter ?